iiod/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ find_library(LIBAIO_LIBRARIES aio)
 find_path(LIBAIO_INCLUDE_DIR libaio.h)
 
 if (LIBAIO_LIBRARIES AND LIBAIO_INCLUDE_DIR)
-	option(ENABLE_AIO "Build IIOD with async. I/O support" ON)
+	option(WITH_AIO "Build IIOD with async. I/O support" ON)
```

Comment:
Does this mean that we only provide the option if the libraries are installed in the system? If so, I'm not sure if it would not be better to always provide the option and look for the libraries if the option is enabled. Then make it explicit that dependencies are needed (cmake failure)... Otherwise, users might not even be aware that this feature is possible...